docs/templates/plugin.rst.j2

```diff
@@ -1,6 +1,11 @@
 :source: @{ source }@
 
+{% if plugin_type == 'module' %}
 .. _@{ module }@:
+{% else %}
+.. _@{ plugin_type }@_@{ module }@:
+{% endif %}
+
 {% for alias in aliases %}
```

Comment:
@abadger Was this bit only for modules?

Reply:
I think we can do aliases of both plugins but currently only have module aliases.  We probably want to put the aliases loop into the modules portion of the conditional or duplicate it for both branches of the conditional.